### PR TITLE
Further format

### PR DESCRIPTION
**Pull request by cjm106**
This branch moves the boxWidthLookup table/logic from the bookSection component into a helper file and introduces a default size of 12 if boxWidth is undefined.

### CODE REVIEW

File: src/components/bookSection/helper/formattingHelper.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+export function BoxWidthConversion(boxWidth: string) {
```
Comment (Gina-Frankel) [inline src/components/bookSection/helper/formattingHelper.tsx:1]: I would change the name of the function to sound more like a function by using a verb - eg:
- `toWidth`
- ` toColWidth` 
- `convertToColWidth`
I thought using col might be good as it stands for column which relates to bootstraps grid system which is what we are using to get the width https://getbootstrap.com/docs/4.1/layout/grid/

Comment (Gina-Frankel) [reply src/components/bookSection/helper/formattingHelper.tsx:1]: Also I think the file name could be more specific, maybe something like 
- colWidthHelper

**Review by Gina-Frankel — COMMENTED** [review]
Looks great - Has made the SectionBox look much cleaner 😄 
Just some naming that could be clearer

File: src/components/bookSection/helper/formattingHelper.tsx (hunk 1 above)
Comment (cjm106) [reply src/components/bookSection/helper/formattingHelper.tsx:1]: I went for something specific for the function name but am open to changing it to something more generic if @jamesAforster agrees.  

With regard to the filename I thought it better to go for a more generic name to give us the option of adding other fomatting options going forward. Happy to go with the group though.

Comment (jamesAforster) [reply src/components/bookSection/helper/formattingHelper.tsx:1]: We could change the filename to colWidthHelper then if we need to add more functionality to it we can broaden it out again?

IMO I think BoxWidthConversion is probably the clearest name in terms of what the function does.

File: src/components/bookSection/bookSection.tsx [hunk 2]
```diff
@@ -76,16 +67,7 @@ export const BookSection = (props: Props) => {
...
-    full: 12,
-    threeQuarter: 9,
-    twoThird: 8,
-    half: 6,
-    third: 4,
-    quarter: 3,
-  }
-
-  const columnWidth = boxWidth && boxWidthLookup[boxWidth]
+  const columnWidth = BoxWidthConversion(boxWidth!)
```
Comment (jamesAforster) [inline src/components/bookSection/bookSection.tsx:70]: what does the exclamation mark do at the end of the argument?

**Review by jamesAforster — APPROVED** [review]

File: src/components/bookSection/helper/formattingHelper.tsx (hunk 1 above)
Comment (Gina-Frankel) [reply src/components/bookSection/helper/formattingHelper.tsx:1]: In terms of BoxWidthConversion can I ask to slightly change it to have a verb in it, just as when I read it at first it was not clear to me that it was a function, so something like  `boxWidthConvert ` or `convertBoxWidth`
- Also can you make it start with a lower-case letter  as functions unless they are components start with lower-case

Comment (Gina-Frankel) [reply src/components/bookSection/helper/formattingHelper.tsx:1]: So the formattingHelper then would store any helper methods related to the styling and layout?
Maybe in terms of the file name maybe it does make sense to keep it generic.  

Is formattingHelper the name that is typically given to such files? Maybe layoutHelper  or styleHelper would work better as it is more specific but still quite general that other functions could be added.

What do @cjm106, @jamesAforster and @benfurber think?

File: src/components/bookSection/bookSection.tsx (hunk 2 above)
Comment (cjm106) [reply src/components/bookSection/bookSection.tsx:70]: It allows boxWidth to be undefined in the markdown - without it the build/tests fail:

 src/components/bookSection/bookSection.tsx:70:42 - error TS2345: Argument of
 type 'string | undefined' is not assignable to parameter of type 'string'.
      Type 'undefined' is not assignable to type 'string'.

    70   const columnWidth = BoxWidthConversion(boxWidth)

File: src/components/bookSection/helper/formattingHelper.tsx (hunk 1 above)
Comment (cjm106) [reply src/components/bookSection/helper/formattingHelper.tsx]: ok, I have changed to the below, please let me know if you're happy with these changes.

helper filename:  colWidthHelper
function: BoxWidthConvert

Comment (Gina-Frankel) [reply src/components/bookSection/helper/formattingHelper.tsx]: 👍

**Pull request closed without merging**